### DISCUSSION TITLE
How to Ask for Help

### DISCUSSION THREAD

**Discussion by drteresavasquez** [nss-evening-cohort-14/evening-client-side#2] [Discuss a Topic]
## RULES: 
You have to answer ALL of the questions below to use the **Help Discussion**
**DO NOT ASK FOR HELP ON ANOTHER DISCUSSION TYPE**
____ 
- Your Name
- Have you searched the other help tickets? **If not, do that first** (Duplicates asks for help will be shamed ceaselessly)
  - If there are similar issues, make sure to give us their number/link to them and tell how yours is different
- Describe your issue
- List any error messages that appear
- What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)
- What have you tried? Tell us everything you have tried. 
- Copy/paste link **TO THE FILE** on your branch or in your repo that you are having issues with NO SCREEN SHOTS 🥰 
**NOTE:** We will not accept links to the whole repo. You must provide a link directly to the specific file

ONCE YOU GET AN ANSWER: mark the answer so that others can get the benefit of seeing the answer. It may very well be that you are asked to go back to your thread and update it with the answer if the instruction team helps you with your issue.

Either way, once you get a resolution either through the discussion or outside of it, you must update with the answer you get.

Did I mention **DO NOT ASK FOR HELP ON ANOTHER DISCUSSION TYPE**??

Give me a comment below letting us know that you understand these items or let us know in the comments if you have any questions.

**Comment by BenJarrett** [comment]
I understand Dr. T!

**Reply by jimguistical** [reply]
I understand how do ask for help!  Help!

**Reply by nathanmartin5937** [reply]
i understand

**Comment by lindseysatterfield** [comment]
Understood!

**Comment by katherinevfry** [comment]
Got it, Dr. T!

**Comment by caseywalker** [comment]
Got it, Dr. T.

**Comment by robertncole1** [comment]
Got it Dr. T!

**Comment by chiestroud** [comment]
I understand!

**Comment by jrobinson0529** [comment]
Roger Roger

**Comment by seancrossettie** [comment]
I got it Dr. T!

**Comment by elliewalkersm** [comment]
Got it!

**Comment by hmparsons-sde** [comment]
heard

**Comment by thedigitalmenagerie** [comment]
Got it!

**Comment by JuanDavila1101** [comment]
I understand

**Comment by saraschoonover** [comment]
Got it!

**Comment by DuleyWilliams** [comment]
I never understood reddit. This seems like the same idea. I get it now.

**Comment by cmeffley** [comment]
Got it!

**Comment by jackiebgregory** [comment]
Understood!

**Comment by nashvegasnate** [comment]
Got it

**Comment by Mitchell-Crumbley** [comment]
Figured it out!

**Comment by danicrosby** [comment]
10-4 Rubber Ducky

**Comment by gage3121** [comment]
I got you Dr. T!

**Comment by tsekeres** [comment]
I'm getting it!

**Comment by GarrettLaw8** [comment]
got it

**Comment by Greenfin17** [comment]
Got it! Thank you.

**Comment by fimoefive** [comment]
Roger that!

**Comment by TaylorHardaway** [comment]
ok I got it!

**Comment by Gabrielle-Tobermann** [comment]
Got it!

**Comment by jimguistical** [comment]
Woot!

**Comment by GonzalesMatthew** [comment]
Hello

**Comment by nathanmartin5937** [comment]
I understand. Thank you.

**Comment by danicrosby** [comment]
10-4 Rubber Ducky
